intern/backend/products.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type Product struct {
+	ID           string `json:"id"`
+	Name         string `json:"name"`
+	Price        int    `json:"price"`
+	Stock		 int    `json:"stock"`
+	ImageURL     string `json:"image_url"`
+}
+
+type ProductsResponse struct {
+	Products []*Product `json:"products"`
+}
+
+var products = []*Product{
+	&Product{
+		ID:           "SHOP-0011",
+		Name:         "Milk",
+		Price:        10,
+		Stock: 20,
+		ImageURL:     "https://images.unsplash.com/photo-1576186726115-4d51596775d1?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80",
```

Comment:
Is the spacing right ? (no space between stock and 20, but space between other key-value pairs 🤔)

Reply:
no I'm installing the formatter